src/components/inputs/Pay/PayInputSubText.tsx

```diff
@@ -125,16 +125,16 @@ export default function PayInputSubText({
     <div
       style={{
         marginTop: '0.3rem',
-        fontSize: '.65rem',
```

Comment:
Bumping this one up a bit are we sure?

Reply:
<img width="1304" alt="Screen Shot 2022-08-25 at 8 19 41 pm" src="https://user-images.githubusercontent.com/96150256/186640012-a42bd353-9720-45f7-8af0-9ac66391dfba.png">

Still looks alright. Up to you